### PR TITLE
Add redirection for webmaker

### PR DESCRIPTION
**Pull request by patjouk**
PR part of the sunset of webmaker mozillafoundation/mofo-devops-private#317

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -233,6 +233,21 @@ class Config(object):
             ReturnCodes.TEMPORARY,
             (True, True)
         ),
+        'www.webmaker.org': (
+            'https://foundation.mozilla.org/en/artifacts/webmaker/',
+            ReturnCodes.PERMANENT,
+            (True, True)
```
Comment (cadecairos) [inline config.py]: Make this tuple and the other two below `(False, True)` so only the query params are preserved

File: config.py [hunk 2]
```diff
@@ -233,6 +233,21 @@ class Config(object):
...
+        ),
+        'beta.webmaker.org': (
+            'https://foundation.mozilla.org/en/artifacts/webmaker/',
+            ReturnCodes.PERMANENT,
+            (True, True)
+        ),
+        'events.webmaker.org': (
+            'https://foundation.mozilla.org/en/artifacts/webmaker/',
+            ReturnCodes.PERMANENT,
+            (True, True)
```
Comment (cadecairos) [inline config.py]:
```suggestion
            (False, True)
        ),
        'beta.webmaker.org': (
            'https://foundation.mozilla.org/en/artifacts/webmaker/',
            ReturnCodes.PERMANENT,
            (False, True)
        ),
        'events.webmaker.org': (
            'https://foundation.mozilla.org/en/artifacts/webmaker/',
            ReturnCodes.PERMANENT,
            (False, True)

```

File: config.py (hunk 1 above)
Comment (cadecairos) [reply config.py]: In order to keep the ball rolling I committed a change suggestion for this

Comment (patjouk) [reply config.py]: Works for me!

**Pull request merged**